core/Dockerfile

```diff
@@ -1,8 +1,12 @@
-FROM openjdk:11.0.15-jdk-buster
+FROM debian:bullseye
 
 RUN apt-get update \
-  && apt-get install git curl -y
+  && apt-get install -y git curl wget software-properties-common unzip \
+  openjdk-17-jdk=17.0.3+7-1~deb11u1 openjdk-17-jre=17.0.3+7-1~deb11u1 \
+  openjdk-17-jdk-headless=17.0.3+7-1~deb11u1 openjdk-17-jre-headless=17.0.3+7-1~deb11u1
 
+
+ENV JAVA_HOME=/usr/lib/jvm/java-17-openjdk-amd64
```

Comment:
we also build ARM image, the build command can be found here:

https://github.com/questdb/questdb/tree/master/artifacts/release

as it stands `amd64` will not work for ARM

Reply:
![image](https://user-images.githubusercontent.com/7276403/172676101-4ef48f1d-591e-435a-859c-ead84b193e7a.png)

Reply:
very good catch. fixing now

Reply:
@bluestreak01 this should be fixed now. Can you try it now?

Reply:
builds fine now, thank you! 🎉